pages/data.py

```diff
@@ -21,7 +21,7 @@
 intro = """## Data"""
 
 layout = html.Div(
-    [   
```

Comment:
this whitespace change reappeared after I did the rebasing

Reply:
thoughts @shankari
Basically my workflow was:
git rebase -i d241d37
edit said commit, removed the whitespace
git add 
git rebase continue
then git push